Milestone5: friend requests

arslanbisharat opened the pull request:
In this model I
>Generated the friendrequest model
>Added validations and associations between models
>Add RSpec tests for each model
>Add integration tests for the site

**I Worked Alone

maelfosso reviewed (approved):
# Congratulations :clap: 

You did a really good job ! :smile: 
You can merge it :+1: 

Good continuation :handshake: 

Mael FOSSO

arslanbisharat committed:
Milestone1: EDR Added

arslanbisharat committed:
Gem updated

arslanbisharat committed:
generate friendrequests model and create model associations ang window gem

arslanbisharat committed:
update friendrequests  model

arslanbisharat committed:
Adds the following

- Installs gem files
- Install rspec and capybara
- adds test for user model

arslanbisharat committed:
adds test for post and friend request models

arslanbisharat committed:
Update tests for post model and add tests for like model

arslanbisharat committed:
Add tests for the comment model Add integration tests for authenticating a user

arslanbisharat committed:
Update factories folder

arslanbisharat committed:
Add integration tests creating a new post and some other setting

arslanbisharat committed:
Add itegration tests for comment and like models

arslanbisharat committed:
linters

arslanbisharat committed:
controller settings

arslanbisharat committed:
Add the following:

- Update the user controller and model
- Update the friend request controller
- Update index and show pages
- Add friends link to the nav bar

arslanbisharat committed:
Add rspec tests for the controllers

arslanbisharat committed:
adds feature test for friend request

arslanbisharat committed:
Add tests to helper methods for the user model

arslanbisharat committed:
linter errors

arslanbisharat committed:
current user reomved

arslanbisharat committed:
linter
